postgresql-persistent/postgres-backup.yaml

```diff
@@ -0,0 +1,23 @@
+- hosts: localhost
+  gather_facts: False
+  tasks:
+    - name: Create a backup
+      tags:
+        - backup
+      k8s:
+        state: present
+        kubeconfig: "{{ lookup('env', 'KUBECONFIG') }}"
+        definition:
+          apiVersion: velero.io/v1
+          kind: Backup
+          metadata:
+            name: postgres-persistent
+            labels:
+              velero.io/storage-location: default
+            namespace: velero
```

Comment:
I would prefer if this was a template so that we can parameterize the velero namespace field.